### PR TITLE
Commits since 0.19.0 branch freeze

### PR DESCRIPTION
**Pull request by awood45**
We were keeping changes since we froze the develop branch for the 0.19.0 release in this branch. Now that 0.19.0 is out, returning these to the develop branch.

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -1,16 +1,16 @@
 six~=1.11.0
 chevron~=0.12
-click~=6.7
+click~=7.0
 enum34~=1.1.6; python_version<"3.4"
 Flask~=1.0.2
 boto3~=1.9, >=1.9.56
-PyYAML~=3.12
+PyYAML~=5.1
```
Comment (milind-shakya-sp) [inline requirements/base.txt]: 👍

**Comment by jfuss** [comment]
And Py2 bites again.... 

`os.symlink` is not available in python2.7's standard library: https://docs.python.org/2.7/library/os.html#os.symlink. This was added in #1140 to address the symlink issue with layers. To unblock this PR, which has a lot of other goodies, I vote to revert #1140 and then take another pass at it separately.

**Comment by bubba-h57** [comment]
Aww man, people still use Windows? :-) Any appetite for a switch, if windows+2.7 then the old method, otherwise, *respect my authorit (symlinks)* :-)

**Comment by jfuss** [comment]
@bubba-h57 Let me have a conversation with the team. I am totally for the switch (mainly because Py2.7 is being removed in Oct as a support Python version), but want to make sure the team is on the same page first. Will respond back here once I have that conversation.

**Comment by jfuss** [comment]
@bubba-h57 The "keep old behavior with Windows+2.7 only" is good. Are you willing to update and push a new PR with this?

**Comment by bubba-h57** [comment]
Certainly. I'll have something updated by the end of the week.

**Pull request merged**